src/components/__tests_/Search.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { screen, render } from '@testing-library/react';
+import Search from '../Search';
+
+it('Renders when there are props passed to it', () => {
+  const tree = renderer
+    .create(<Search nameState="name state" handleOnChange={() => ''} />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Kindly fix the failing test here.
![image](https://user-images.githubusercontent.com/81932478/194724573-8b4acb29-5336-4a75-9a3e-9e74ef6c22ed.png)